### PR TITLE
Added `--local` option (for loading active learning/damon models)

### PR DESCRIPTION
**Pull request by ZechyW**
When a model is saved in active learning mode, it resumes active learning when loaded (i.e., it starts listening for input on a port and ignoring local sources).
This option allows the user to change this behaviour (so that the Python library can interact with the loaded model, for example).

### CODE REVIEW

File: vowpalwabbit/global_data.h [hunk 1]
```diff
@@ -525,7 +525,9 @@ struct vw
   VW_DEPRECATED("This is unused and will be removed")
   char* program_name;
 
-  bool stdin_off;
+  bool stdin_off;
+
+  bool local;  // If a model was saved in daemon or active learning mode, start it in local mode instead
```
Comment (jackgerrits) [inline vowpalwabbit/global_data.h:530]: This needs to be initialized with a default value

File: vowpalwabbit/parse_args.cc [hunk 2]
```diff
@@ -438,6 +438,7 @@ input_options parse_source(vw& all, options_i& options)
                   "use gzip format whenever possible. If a cache file is being created, this option creates a "
                   "compressed cache file. A mixture of raw-text & compressed inputs are supported with autodetection."))
       .add(make_option("no_stdin", all.stdin_off).help("do not default to reading from stdin"))
+      .add(make_option("local", all.local).help("Do not start in daemon or active learning mode (for loaded models)"))
```
Comment (jackgerrits) [inline vowpalwabbit/parse_args.cc:441]: Can you please make the option name more specific, so that it's very clear to users it only affects daemon/active

Comment (ZechyW) [reply vowpalwabbit/parse_args.cc:441]: Got it, how do you feel about `--force-local-input` or `--no-daemon`?

Comment (jackgerrits) [reply vowpalwabbit/parse_args.cc:441]: I'd be okay with `--no-daemon`

**Comment by ZechyW** [comment]
Have made the requested changes -- Thanks for checking, @jackgerrits, and let me know if you need me to modify it any further!

**Comment by ZechyW** [comment]
@peterychang: Didn't see any other way to revert to local input for an active learning model once it's saved, but let me know if there was already an option for it or if there's a better way of doing it!

**Comment by peterychang** [comment]
I agree. I just find it silly that we actually need this change. IIRC @lokitoth you were doing some work to allow alteration of the model-encoded options, right? Would that work help with this?

**Comment by ZechyW** [comment]
A general way to change the saved model options sounds very useful indeed -- Will keep an eye out for it.  Thanks for the update!

**Pull request merged**